src/api.js

```diff
@@ -40,4 +40,14 @@ export function getComments(article_id) {
     .catch((err) => {
         console.log(err)
     })
+}
+
+export function updateArticleVotes(article_id, votes) {
+    return axios.patch(`https://be-nc-news-q0tg.onrender.com/api/articles/${article_id}`, {inc_votes: votes})
+    .then((response) => {
+        console.log(response)
+    })
+    .catch((err) => {
+        console.log(err)
```

Comment:
If the user is offline and tries to up or down vote they currently don't get any feedback to let them know this patch hasn't worked.